sdk/monitor/Azure.Monitor.Query/src/LogsQueryOptions.cs

```diff
@@ -2,12 +2,25 @@
 // Licensed under the MIT License.
 
 using System;
+using Azure.Monitor.Query.Models;
 
 namespace Azure.Monitor.Query
 {
+    /// <summary>
+    /// Options for <see cref="LogsClient.QueryAsync"/> that allow specifying the service timeout or whether to include
+    /// the query execution statistics.
```

Comment:
Should we stay more general here like:
```suggestion
    /// Options for <see cref="LogsClient.QueryAsync"/>.
```

and let the option properties speak for themselves? Just thinking that if these expand over time, that we would need to keep adding to this description.